### ISSUE TITLE
Jet Woo Builder Layout Switcher doesn't apply CSS on defined pages for layout changer

### ISSUE THREAD

**Issue by ivntrdata** [CrocoBlock/suggestions#2825]
**Describe the bug**
I've created two layouts - a grid and a list style for my shop as woobuilder templates. I've selected the main layout and the secondary layout to be those Woobuilder templates. Each layout has had changes to the styling of the widgets using the widget options. However when trying to toggle between the two layouts on the store page it's not applying the CSS after the ajax update. The layout changes but the changed styles don't apply and I have to reload the page for them to apply. At the very least it would be nice to have an option to do a page reload for the layout change... you've got this as an option for smartfilters, why not the layout changer? 

Using the Hello Elementor theme

**Comment by UraraReika** [comment]
Hello @ivntrdata !
Could you please contact our support team so we can check the issue on your site?

**Comment by ivntrdata** [comment]
I did contact support. I keep getting bounced around. Used the live chat, they said submit a support ticket, I submitted a support ticket they said to post on github, now you're saying to contact support again?

Does anyone there actually DO support or is it just a bunch of people getting paid to pass the problem on to someone else? Getting a little frustrated with being bounced around going on close to a week and only finding more problems and limitations with your software each day.
